src/main/java/com/auth0/client/auth/AuthAPI.java

```diff
@@ -408,6 +410,39 @@ public AuthRequest requestToken(String audience) {
     }
 
     /**
+     * Creates a new request to revoke an existing Refresh Token.
+     * <pre>
+     * {@code
+     * AuthAPI auth = new AuthAPI("me.auth0.com", "B3c6RYhk1v9SbIJcRIOwu62gIUGsnze", "2679NfkaBn62e6w5E8zNEzjr-yWfkaBne");
+     * try {
+     *      auth.revokeToken("ej2E8zNEzjrcSD2edjaE")
+     *          .execute();
+     * } catch (Auth0Exception e) {
+     *      //Something happened
+     * }
+     * }
+     * </pre>
+     *
+     * @param refreshToken the refresh token to revoke.
+     * @return a Request to configure and execute.
+     */
+    public Request<Void> revokeToken(String refreshToken) {
+        Asserts.assertNotNull(refreshToken, "refresh token");
+
+        String url = HttpUrl.parse(baseUrl)
```

Comment:
totally unrelated: wasn't this repo that I made a PR to avoid having to parse each time? also this can fail, so it's better to fail fast, right in the constructor..